tests/phpunit/utils/Traits/With_Mock_Filesystem.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Trait With_Mock_Filesystem.
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Test_Utils\Traits;
+
+trait With_Mock_Filesystem {
+	/**
+	 * Sets up a Mock Filesystem.
+	 *
+	 * @since n.e.x.t
+	 */
+	protected function set_up_mock_filesystem() {
+		global $wp_filesystem;
+
+		add_filter(
+			'filesystem_method_file',
+			function() {
+				return __DIR__ . '/../testdata/Filesystem/WP_Filesystem_MockFilesystem.php';
```

Comment:
```suggestion
				return TESTS_PLUGIN_DIR . '/testdata/Filesystem/WP_Filesystem_MockFilesystem.php';
```

Reply:
I don't like path like this.

Reply:
Thanks @spacedmonkey this has been updated